src/PowerShellEditorServices/Services/Analysis/AnalysisService.cs

```diff
@@ -766,151 +361,53 @@ private class PowerShellResult
             // on.  It makes sense to send back the results from the first
             // delay period while the second one is ticking away.
 
-            // Get the requested files
             foreach (ScriptFile scriptFile in filesToAnalyze)
             {
-                List<ScriptFileMarker> semanticMarkers = null;
-                if (isScriptAnalysisEnabled)
-                {
-                    semanticMarkers = await GetSemanticMarkersAsync(scriptFile).ConfigureAwait(false);
-                }
-                else
-                {
-                    // Semantic markers aren't available if the AnalysisService
-                    // isn't available
-                    semanticMarkers = new List<ScriptFileMarker>();
-                }
+                ScriptFileMarker[] semanticMarkers = await AnalysisEngine.AnalyzeScriptAsync(scriptFile.Contents).ConfigureAwait(false);
 
                 scriptFile.DiagnosticMarkers.AddRange(semanticMarkers);
 
-                PublishScriptDiagnostics(
-                    scriptFile,
-                    // Concat script analysis errors to any existing parse errors
-                    scriptFile.DiagnosticMarkers);
+                PublishScriptDiagnostics(scriptFile);
             }
         }
 
-        internal void ClearMarkers(ScriptFile scriptFile)
-        {
-            // send empty diagnostic markers to clear any markers associated with the given file
-            PublishScriptDiagnostics(
-                    scriptFile,
-                    new List<ScriptFileMarker>());
-        }
+        private void PublishScriptDiagnostics(ScriptFile scriptFile) => PublishScriptDiagnostics(scriptFile, scriptFile.DiagnosticMarkers);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'PublishScriptDiagnostics' a static method.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4961943)

Reply:
The method this points to is an instance method...